src/model/nets/models_genesis.py

```diff
@@ -21,7 +22,7 @@ class ModelsGenesisSegNet(BaseNet):
         frozen_modules (sequence, optional): Specify which modules should be frozen (default: None).
     """
 
-    def __init__(self, in_channels, out_channels, weight_path=None, loaded_modules=None, frozen_modules=None):
+    def __init__(self, in_channels, out_channels, weight_path=None, loaded_modules=None, frozen_modules=None, weight_settings=None, norm_trainble_only=False):
```

Comment:
Please update the comments for args `weight_settings` and `norm_trainble_only`, thanks.

Reply:
Updated.